cscs-checks/prgenv/helloworld.py

```diff
@@ -84,14 +95,15 @@ def num_ranks(match):
             }
         }
 
-        self.maintainers = ['VH', 'EK']
-        self.tags = {'production', 'craype'}
-
     @run_before('compile')
     def setflags(self):
```

Comment:
I'd suggest something like this and you could merge it with the other hook setting the linking type.

```suggestion
    def prepare_build(self):
```